docs/examples/template-old.csv

```diff
@@ -0,0 +1,5 @@
+ID,Label,Definition,Definition Source,See Also,Editor,RDF Type,Class Type,Parent IRI

```

Comment:
Renaming this file is fine, but we need the test to run somewhere. Now I think it is not.